src/ol/interaction/Select.js

```diff
@@ -291,6 +291,15 @@ class Select extends Interaction {
     return this.hitTolerance_;
   }
 
+  /**
+   * Returns the overlay layer that stores selected features.
+   * @return {module:ol/layer/Vector} Layer.
+   * @api
+   */
+  getFeatureOverlay() {
```

Comment:
Would it really be necessary to expose the featureOverlay? If it is managed by the user, there is already a reference somewhere. Since it was not exposed before, I  think it does not have to be exposed now.

Reply:
Ah good catch, this was from an earlier draft. I'll remove it.